### PR TITLE
[14.0][MIG] purchase_picking_state: Migration to 14.0

### PR DESCRIPTION
**Pull request by victoralmau**
Superseed: https://github.com/OCA/purchase-workflow/pull/1157

Migration to 14.0

Please @pedrobaeza and @ernestotejeda can you review it?

@Tecnativa TT33424

### CODE REVIEW

**Comment by kakurai8** [comment]
I have signed the CLA and my application is in process. Please wait.

**Review by ernestotejeda — COMMENTED** [review]
Code review: 👍🏼

**Review by ernestotejeda — APPROVED** [review]

File: purchase_picking_state/models/purchase.py [hunk 1]
```diff
@@ -20,19 +20,17 @@ def get_picking_state(self):
...
-                if pickings_state == {"cancel"}:
-                    purchase.picking_state = "cancel"
-                elif pickings_state == {"cancel", "done"} or pickings_state == {"done"}:
-                    purchase.picking_state = "done"
-                elif "done" in pickings_state:
-                    purchase.picking_state = "partially_received"
-                else:
-                    purchase.picking_state = "not_received"
+        self.picking_state = "draft"
+        for purchase in self.filtered("picking_ids"):
```
Comment (pedrobaeza) [inline purchase_picking_state/models/purchase.py]: Better to leave previous code for not having to pass 2 times through the recordset.

**Comment by pedrobaeza** [comment]
Please check pre-commit stuff with tabs.

/ocabot migration purchase_picking_state

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**